evolver/logutils.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+EVENT = logging.INFO + 1
+logging.addLevelName(EVENT, "EVENT")
+
+
+class LogHistoryCaptureHandler(logging.Handler):
+    def __init__(self, history_server):
+        super().__init__()
+        self.history_server = history_server
+
+    def emit(self, record):
+        vial = getattr(record, "vial", None)
+        record_dict = {"level": record.levelname, "message": record.getMessage()}
+        # TODO: add support for some extra fields (but these should be json-seralizable)
+        kind = "event" if record.levelno == EVENT else "log"
```

Comment:
[comment] It might help the TODO implementer if we added here and now, how we might envision coding this, e.g., using a [``match``](https://docs.python.org/3/tutorial/controlflow.html#match-statements), an ``enum``, a mapping etc?